LICENSES/AGPL-3.0-or-later.txt

```diff
@@ -0,0 +1,214 @@
+GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Do we need to add this license only for the Nextcloud logo?

Reply:
As we have discussed by email, the issue here is resumed by the trademark usage under Nextcloud's trademark policy. So no need for this license here.